crates/bevy_ui/src/ui_node.rs

```diff
@@ -2266,6 +2266,39 @@ impl ResolvedBorderRadius {
     };
 }
 
+#[derive(Component, Copy, Clone, Debug, PartialEq, Reflect)]
+#[reflect(Component, PartialEq, Default)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+pub struct BoxShadow {
```

Comment:
Needs doc comments with a brief description and an explanation of how it's used. In particular, I want to know if this goes on my Node component, or as a child, and any limitations.

I'd also like to have a note about what `Val`'s dimensions are computed relative to.